src/components/select/filterable-select/index.js

```diff
@@ -0,0 +1 @@
+export { default } from './filterable-select.component';
```

Comment:
We're exposing two ways of using filterable-select.

- `import {FilterableSelect} from 'carbon-react/lib/components/select`
- `import FilterableSelect from 'carbon-react/lib/components/select/filterable-select`

We should have only one way to do this.

Reply:
it's against our `Prefer default export` linting rule. We need to either remove that rule to avoid use of `disable rule` comments or we leave it as it is. I think that it's pretty clear in the documentation how this component should be imported.

Reply:
Sorry for the confusion, I mean we shouldn't allow the component to be imported from two different locations.

We should either delete the file `src/components/select/filterable-select/index.js`.
Or we should remove the named import `FilterableSelect` from `carbon-react/lib/components/select`.

Reply:
file deleted

Reply:
The file is still here.

Reply:
Unresolved.

Reply:
I'm sorry, I must have missed it